projects/ngx-mask-lib/src/lib/ngx-mask.directive.ts

```diff
@@ -614,12 +614,13 @@ export class NgxMaskDirective implements ControlValueAccessor, OnChanges, Valida
                 }
                 // update position after applyValueChanges to prevent cursor on wrong position when it has an array of maskExpression
                 if (this._maskExpressionArray.length) {
-                    if (this._code === MaskExpression.BACKSPACE) {
-                        position = this.specialCharacters.includes(
-                            this._inputValue.slice(position - 1, position)
+                    if (
+                        this._code === MaskExpression.BACKSPACE &&
+                        this.specialCharacters.includes(
```

Comment:
could we use some variable - the main reason do not have big expressions in conditions